### ISSUE TITLE
Improve README

### ISSUE THREAD

**Issue by taoeffect** [taoeffect/empress#11]
1. Someone who has no idea what Ansible is should be able to quickly get up and running.
2. It should be clearly spelled out what files should be edited.
3. It should be clearly spelled out what Empress will do to someone's system.
4. We should link to our own wiki.
##

**Comment by taoeffect** [comment]
Ideally, zero knowledge or use of Ansible should be required. cc @PiPeep

**Comment by NJAldwin** [comment]
Might I suggest adding some sort of a roadmap?  As someone who doesn't know very much about these things, but who'd like to set up empress, I have no idea how long I should wait until it is stable enough for me to use.  Things like "Self-signed certs are free, easy, but are not yet authenticated by TLS (but will be)." don't really give a good indication (to me, at least) of when things might be fixed, and thus how long I should hold off on looking into setting up empress.

**Comment by taoeffect** [comment]
> Things like "Self-signed certs are free, easy, but are not yet authenticated by TLS (but will be)." don't really give a good indication (to me, at least) of when things might be fixed

Hmm, that line apparently confuses folks, so maybe I should get rid of that (or completely rephrase it).

That particular statement is a statement about the state of TLS in general (on the Internet), not about any issue with this project.

As far a road map goes, however... yes, that would be useful, but the truth is that this project is being worked on by only two people currently (myself and @PiPeep), and we do this in our spare time and for zero compensation (so far at least :P).

So in other words I think a roadmap at this point would be a little premature and not super helpful to folks.

**Comment by taoeffect** [comment]
(That said, you are welcome to help us test and improve Empress. It's just suitable for sysadmins only at this point in time.)

**Comment by bgw** [comment]
@NJAldwin, I concur with everything @taoeffect said. Since the original purpose of this project (provide @taoeffect with a mail-server) is served, I'm currently more focused on [ansible-honeybadger](https://github.com/pipeep/ansible-honeybadger), as I think it has the potential for more immediate positive social impact.

That said, if there's a serious issue, we'll patch it, but expect a lead time of a couple days.

If you're asking for a roadmap because you'd like to contribute, any issue with the `low-hanging fruit` tag is a good place to start.
